AEXS/aex-3.md

```diff
@@ -0,0 +1,71 @@
+# AEX-3
+
+```
+AEX: 3
+Title: Secret storage format
+Author: Sascha Hanse <shanse@posteo.de>, Shubhendu Shekhar (@shekhar-shubhendu)
+License: BSD-3-Clause
+Status: Draft
+Type: Informational
+Created: 2019-04-03
+```
+
+## Simple Summary
+
+The document describes and defines the secret storage approach used by æternity.
+
+## Motivation
+
+The motivation for the AEX is to describe a standard way that is being used by æternity for secret storage.
+
+The secret storage specification, although is being currently used for secret-key storage, should not be limited to it and should be used as a standard way of storing any data (esp. user related or user-owned) in an encrypted format.
```

Comment:
I'm not sure if this should be used to store any data. Do we really want big binaries in this?

Reply:
@knarz i was thinking more for text(instead of big binaries). there was a use case where third-party wallets can provide methods to store user secrets. I'll clarify it in the document too.